index.html

```diff
@@ -80,6 +80,216 @@ <h2 class="purple">LET'S CONNECT</h2> <br>
+                                </svg>  
+                            </a>
+                        </li>
+                    </ul>               
+                   </div>
+                   <button class="btn resume-btn" type="button">Get my resume</button>
+                   <ul class="about-list">
+                      <li><h3 class="languages-title">Languages</span></h3></li> <br>
+                      <div class="langs">
+                            <ul>
+                                <li>
+                                    <span class="language-container">
+                                        <img src="images/js-logo.png" id="js" alt="JS logo">                                   
+                                        <p>JavaScript</p>
+                                    </span>
+                                </li> <br>
+                                <li>
+                                    <span class="language-container">
+                                        <img src="images/html-logo.png" alt="HTML logo">                                   
+                                        <p>HTML</p>
+                                    </span>
+                                </li> <br>
+                                <li>
+                                    <span class="language-container">
+                                        <img src="images/css-logo.png" alt="CSS logo">                                   
+                                        <p>CSS</p>
+                                    </span>
+                                </li>
+                            </ul>
+                      </div>
+                      <li><h3 class="languages-title">Frameworks</h3></li> <br>
+                      <li><h3 class="languages-title">Skills    </h3></li>
+                   </ul>
```

Comment:
- [ ] As stated in the project requirement You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). but for For each language  the image are broken as shown below in screenshot , the path you use for image `<img src="images/js-logo.png" id="js" alt="JS logo">` has no image for js-logo.png, html-logo.png, css-logo.png in your image folder, Kindly Fix it.
- Your Image Folder 
![Capture](https://user-images.githubusercontent.com/93598189/235237860-75bbe49b-4857-4196-9fba-2009c1c9c82e.PNG)

- Your Project technology categories Design Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/235239477-859ceb67-06ae-4119-814a-2f38f07c3402.PNG)

-  Figma Project technology categories Design Look likes this : 
![Capture](https://user-images.githubusercontent.com/93598189/235239826-1e50fad8-3d3f-4ee0-b216-7c0625fd2f8a.PNG)